dohq_artifactory/admin.py

```diff
@@ -11,6 +12,19 @@ def rest_delay():
     time.sleep(0.5)
 
 
+def raise_errors(r):
+    try:
+        r.raise_for_status()
+    except requests.HTTPError as e:
+        if e.response.status_code >= 400:
+            logging.warning("Artifactory returned an error. Response text: {}".format(e.resonse.text))
```

Comment:
A misprint is here - `resonse`

May be we could add some unit-test for this function?)

Reply:
Added under `tests/integration/unit/test_exceptions.py`.

Created a separate file because maybe as more tests will get added around exception handling we can use this, but I'm open to moving it to a more appropriate location.